test/source/tests/setup.ts

```diff
@@ -787,6 +787,34 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
       expect(savedPassphrase).to.be.an('undefined');
     }));
 
+    ava.default('user@passphrase-session-length-client-configuration.flowcrypt.test - passphrase should expire in in_memory_pass_phrase_session_length', testWithBrowser(undefined, async (t, browser) => {
+      const acctEmail = 'user@passphrase-session-length-client-configuration.flowcrypt.test';
+      const longid = '715EDCDC7939A8F7';
+      const settingsPage = await BrowserRecipe.openSettingsLoginApprove(t, browser, acctEmail);
+      const passphrase = '1234';
+      await SetupPageRecipe.manualEnter(settingsPage, 'unused', {
+        submitPubkey: false,
+        usedPgpBefore: false,
+        key: {
+          title: 'my key',
+          armored: testConstants.testkey715EDCDC7939A8F7,
+          passphrase,
+          longid,
+        }
+      });
+      const { cryptup_userpassphrasesessionlengthclientconfigurationflowcrypttest_rules: rules
+      } = await settingsPage.getFromLocalStorage([
+        'cryptup_userpassphrasesessionlengthclientconfigurationflowcrypttest_rules'
+      ]);
+      let savedPassphrase = await BrowserRecipe.getSavedPassPhrase(settingsPage, acctEmail, longid);
+      expect((rules as { in_memory_pass_phrase_session_length: number }).in_memory_pass_phrase_session_length).to.be.equal(60);
```

Comment:
The test is correct.
Should we really wait for 60 seconds?
Can we set up expiration to 5 or 10 seconds, since you defined `min: 1` second?

Reply:
I was also thinking this

Reply:
Or in addition to the 60 second timer (which would be a fallback) we could have a `Date.now() + expirationSeconds` timeout - just a single run, that will clear it immediately. That way, if you set 1 second timeout, you could test two seconds later.

Reply:
> Maybe current app code has a 60 second timer (setInterval) for removing pass phrases? We could lower the interval, and have a shorter test.

For removing -- maybe. But there is also an on-demand (post-search) removal in `ExpirationCache.get`.
And from the caller's point of view these two cases can't be distinguished.

Reply:
Could that be made pre-search instead? Then the test could test immediately post expiry.

Reply:
I meant -- after search but before (validating and) returning the result.
So we can test immediately after expiry, no changes should be needed.